include/rocksdb/c.h

```diff
@@ -2790,5 +2789,5 @@ extern ROCKSDB_LIBRARY_API void rocksdb_disable_manual_compaction(
 extern ROCKSDB_LIBRARY_API void rocksdb_enable_manual_compaction(rocksdb_t* db);
 
 #ifdef __cplusplus
-}  /* end extern "C" */
+} /* end extern "C" */
```

Comment:
Changing this line to `}  // extern "C"` would look more normal to me

Reply:
Perhaps a follow up:)